packages/components/psammead-copyright/README.md

```diff
@@ -1,19 +1,52 @@
 # psammead-copyright &middot; [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://bbc-news.github.io/psammead/?selectedKind=Copyright) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-copyright.svg)](https://www.npmjs.com/package/@bbc/psammead-copyright) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
 
-### Description
-Displays the source attribution of an image in block capitals in the bottom-right of the parent element. 
+## Description
+
+Displays a source attribution in block capitals in the bottom-right of the parent element. This implementation is primarily intended for use alongside images.
+
+## Installation
+
+`npm install @bbc/psammead-copyright`
+
+## Props
+
+| Argument  | Type                | Required | Example         |
+|-----------|---------------------|----------|-----------------|
+| No props. |
```

Comment:
Table structure

Actually, wouldn't we just delete the table altogether and just have a sentence "No props"?